Homework3/Task3.py

```diff
@@ -0,0 +1,25 @@
+"""Task 3 from Homework3 (dictionaries)"""
+
+
+def generate_numbers(n=20):
+
+    return dict({key: key**2 for key in range(n + 1)[1:]})
```

Comment:
1. please read range function docs - we don't need slice here we can make necessary numbers within one range
2. in dict({.....}) either {} or dict are redundant